SEED-platform/seed
# Strict table mappings

#### What's this PR do?
During mapping, if columns are mapped to both the Property and Tax Lot tables then require at least one matching field from each table
#### How should this be manually tested?
- Upload a file
- Set all columns to `Property`.  There should be an error only if at least one of the 4 required Property fields is not set (Address Line 1, Custom Id 1, Pm Property Id, Ubid)
- Set all columns to `Tax Lot`.  There should be an error only if at least one of the 3 required Tax Lot fields is not set (Address Line 1, Custom Id 1, Jurisdiction Tax Lot Id)
- Set both `Property` and `Tax Lot` columns.  There should be an error if a mapping field for each table is not set.
#### What are the relevant tickets?
#1587
